access_time.cmake

```diff
@@ -1,12 +1,12 @@
 
 
-set(POLICY_NAME "apply-access-time")
+set(POLICY_NAME "apply_access_time")
 
-set(IRODS_PACKAGE_COMPONENT_POLICY_NAME "${POLICY_NAME}")
+set(IRODS_PACKAGE_COMPONENT_POLICY_NAME "apply-access-time")
```

Comment:
can this be a derived hyphenated version of POLICY_NAME?
aka `string(REPLACE "_" "-" POLICY_NAME_HYPHENS ${POLICY_NAME})`